app/models/concerns/user_multifactor_methods.rb

```diff
@@ -61,11 +69,17 @@ def strong_mfa_level?
     end
 
     def mfa_recommended?
-      return false if strong_mfa_level?
+      return false if strong_mfa_level? || mfa_required?
```

Comment:
Hmm, if we add `|| mfa_required?`, owners without the mfa_required cookie and have more than 180 downloads will not be recommended to have MFA if we ship this.

Reply:
This code is actually live already, just an artifact of being having branched off a rebased/merged branch:

https://github.com/rubygems/rubygems.org/blob/master/app/models/concerns/user_multifactor_methods.rb#L72

Reply:
Isn't it fine for 180+ to not see recommended? They're going to be required in 6 days soon.

Reply:
Ah yeahh... nvm then. It should be fine 👍 
Could you rebase off of the current master so it's easier to see the most up to date changes?